compute.py

```diff
@@ -0,0 +1,8 @@
+""" Module to compute stuff
+"""
+
+
+def my_sum(x, y):
+    """ Compute the sum of 2 numbers
+    """
+    return (x + y) #RD

```

Comment:
You don't need to pyt your initials in the comments. git will remember forever who contributed what changes to what file/line.